src/main/java/stepsdesign/ControlCenterSteps.java

```diff
@@ -0,0 +1,205 @@
+    CdfControlCenterActions.enterValueInput(value);
+  }
+
+  @Then("Verify user is able to click on save and close button of set preferences")
+  public void clickSaveAndCloseButton() {
+    CdfControlCenterActions.clickOnSaveAndCloseButton(); }
+
+  @Then("Verify user is able to click on the data pipeline added in the control center page")
+  public void clickOnDataPipelineTab() {
+    CdfControlCenterActions.clickOnDataPipelineTab(); }
+
+  @Then("Verify user is able to click on the delete icon of preferences to delete the added preferences successfully")
+  public void clickOnDeleteIconSetPreferences() {
+    CdfControlCenterActions.clickOnDeleteIconSetPreferences(); }
+
+  @Then("Verify user is able to click on the dataset entity icon to see the details page of the dataset")
+  public void clickOnDatasetEntityIcon() {
+    CdfControlCenterActions.clickOnDatasetEntityIcon(); }
+
+  @Then("Verify user is navigated to the details page of the dataset entity icon successfully")
+  public void navigateDetailsPageDatasetEntity() {
+
+    CdfControlCenterActions.navigateDetailsPageDatasetEntity();
+  }
+
+  @Then("Click on the delete icon of the dataset entity and the dataset should get deleted successfully")
+  public void deleteDatasetEntity() {
+    CdfControlCenterActions.clickOnDeleteDatasetEntity(); }
+
+  @Then("Verify the deleted dataset {string} entity is not present in control center page")
+  public void deletedDatasetIsNotPresent(String value) {
+    CdfControlCenterActions.deletedDatasetNotPresent(value);
+  }
+
+  @Then("Click on the truncate icon of the dataset entity and the dataset should get truncated successfully")
+  public void truncateDatasetEntity() {
+    CdfControlCenterActions.clickOnTruncateDatasetEntity(); }
+
+  @Then("Click on the view details tab of dataset entity page to see the details page of the dataset")
+  public void clickOnViewDetailsPageDataset() {
+    CdfControlCenterActions.clickOnViewDetailsPageDataset(); }
+
+  @Then("Verify user is successfully navigated to details page of the dataset entity")
+  public void navigatedToViewDetailsPageDataset() {
+    CdfControlCenterActions.navigateToViewDetailsPageDataset(); }
+
+  @Then("Click on the back link of the view details page of dataset entity")
+  public void clickBackTabDatasetEntity() {
+    CdfControlCenterActions.clickOnBackTabDatasetEntity(); }
+
+  @Then("Click on the close link of the view details page of dataset entity to close the details page")
+  public void clickCloseTabDatasetEntity() {
+    CdfControlCenterActions.clickOnCloseTabDatasetEntity(); }
+
+  @Then("Select dropdown : {string} with option value: {string} in control center")
+  public void selectDropdownFilterOptionValue(String pluginProperty, String option) {
+    CdfControlCenterActions.selectFilterDropdownOption(pluginProperty, option); }
+
+  @Then("Verify user is able to see all entities displayed message on the basis of filter selection")
```

Comment:
Verify the all entities message is displayed with the filter selection

Reply:
changed